src/containers/InteractiveMessage/InteractiveMessage.tsx

```diff
@@ -236,7 +236,8 @@ export const InteractiveMessage = () => {
 
     setTitle(titleText);
     setFooter(data.footer || '');
-    setBody(data.body);
+    setBody(data.body || '');
```

Comment:
We could definitely do something better for this. This looks hacky. Fine for now as its breaking on prod. Can we create an issue for this to tackle later.

Reply:
created an issue here #2834